app/controllers/registrations_controller.rb

```diff
@@ -1,10 +1,9 @@
 class RegistrationsController < Devise::RegistrationsController
   prepend_before_action :require_no_authentication, only: []
-  # No authorization required for public registration route
 
   def new
     if user_signed_in?
-      redirect_to "/dashboard?signed-in-already&t=#{Time.current.to_i}"
```

Comment:
Removed it, do we actually need it? I can't find usage for `signed-in-already` and the `t` parameter anywhere